.devcontainer/devcontainer.json

```diff
@@ -1,18 +1,36 @@
 {
-    "name": "Azure Developer CLI",
-    "build": {
-        "dockerfile": "Dockerfile",
-        "args": {
-            "IMAGE": "bullseye"
-        }
+    "name": "Azure Developer CLI - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "1.21"
+        },
+        "ghcr.io/devcontainers/features/docker-in-docker:1": {
+            "version": "latest",
+            "moby": true
+        },
+        "ghcr.io/devcontainers/features/azure-cli:1": {},
+        "ghcr.io/devcontainers/features/dotnet:2": {},
+        "ghcr.io/devcontainers/features/java:1": {},
+        "ghcr.io/devcontainers/features/node:1": {},
+        "ghcr.io/devcontainers/features/python:1": {},
+        "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {},
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/terraform:1": {}
+        
     },
     "customizations": {
         "vscode": {
             "extensions": [
                 "redhat.vscode-yaml",
                 "streetsidesoftware.code-spell-checker",
-                "golang.go"
+                "golang.go",
+                "ms-azuretools.vscode-bicep",
+                "eamodio.gitlens",
+                "hashicorp.terraform"
             ]
         }
-    }
+    },
+    "postAttachCommand": "curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.54.2"
```

Comment:
Could we not use [https://github.com/guiyomh/features/blob/main/src/golangci-lint/README.md](ghcr.io/guiyomh/features/golangci-lint)]

```
"features": {
    "ghcr.io/guiyomh/features/golangci-lint:0": {}
}
````

Reply:
For some reason I was only looking at `ghcr.io/devcontainers`.  I'm happy to use the feature!!
Updated.
Thank you!!